GOVERNANCE.md

```diff
@@ -1,132 +1,175 @@
 # OpenRefine Governance Model
 
 ## Summary / Overview
-OpenRefine is a free, open-source, powerful tool for working with messy data. OpenRefine has a plugin architecture and is distributed under the [new BSD license](http://opensource.org/licenses/BSD-3-Clause) allowing modification, distribution and name changes.
+OpenRefine is a free, open-source, powerful tool for working with messy data. OpenRefine has a [plugin architecture](https://openrefine.org/docs/technical-reference/writing-extensions) and is distributed under the [new BSD license](http://opensource.org/licenses/BSD-3-Clause) allowing modification, distribution and name changes.
 
 ## Roles and responsibilities
-OpenRefine development is based on user consensus and open discussion between users. Decision making must be done in a transparent, open fashion (i.e. using OpenRefine's discussion list and issue list). No decisions about the project’s direction, bug fixes or features may be done in private without community involvement and participation. Discussions must begin at the earliest possible point on a topic; the community’s participation is vital during the entire decision-making process.
+OpenRefine is a meritocratic, consensus-based community project based on the [Apache Decision Making process](https://community.apache.org/committers/decisionMaking.html).
 
-All project participants abide by the [Code of Conduct](https://github.com/OpenRefine/OpenRefine/blob/master/CODE_OF_CONDUCT.md).
+Decision-making is done based on user consensus following open discussion (i.e., using OpenRefine's [discussion list](https://forum.openrefine.org/c/dev/8) and [issue list](https://github.com/OpenRefine/OpenRefine/issues?state=open)). No decisions about the project’s direction, bug fixes, or features may be made in private without community involvement and participation. Discussions must begin at the earliest possible point on a topic; the community’s participation is vital during the entire decision-making process.
```

Comment:
> No decisions about the project’s direction, bug fixes, or features may be made in private without community involvement and participation.

This is not really align with our practice or the Apache Decision Making process. There are a lot of reasons to decide certain things in private. Security, actual consensus making, etc. If we want to keep it simple lets just link to the Apache document without trying to patch it?